src/ray/util/logging.cc

```diff
@@ -114,6 +114,8 @@ void RayLog::ShutDownRayLog() {
 #endif
 }
 
+bool RayLog::IsLevelEnabled(int log_level) { return log_level >= severity_threshold_; }
```

Comment:
const specifier

Reply:
Complier complains "static member function cannot have 'const' qualifier".

Reply:
Oops, I didn't see it.